src/components/titles/index.js

```diff
@@ -160,12 +193,34 @@ function draw(gd, titleClass, options) {
             lineposition: fontLineposition,
         })
         .attr(attributes)
-        .call(svgTextUtils.convertToTspans, gd);
+            .call(svgTextUtils.convertToTspans, gd);
+
+        var subtitleAttributes = Object.assign({}, attributes);
+        subtitleAttributes.y += Drawing.bBox(titleEl.node()).height;
```

Comment:
I think, when the `title.text` is mathjax this calculation is not correct.

Reply:
@emilykl this is no longer a problem. Right?

Reply:
@archmoj Fixed in most cases, but there's still a few edge cases where the position is wrong -- for example, after editing the subtitle, it jumps back up to the wrong position. Working on it.

Reply:
> there's still a few edge cases where the position is wrong -- for example, after editing the subtitle, it jumps back up to the wrong position.

This bug is resolved by [e70e521](https://github.com/plotly/plotly.js/pull/7012/commits/e70e521fac8d59c914d7c8ad208f6efa5fe5d705)